raw_data/preprocessed/africa/seychelles_english.txt

```diff
@@ -0,0 +1,91 @@
+
+(30) [1] Any person who commits an offence under section 16[2] or section 17[5] or section 22[2] or section 22[3] is liable on conviction, to imprisonment for one year and a fine of R50.000 and, if the offence is continued after conviction, is liable to a further fine of R5.000 for each day during which the offence is so continued. [2] A person who commits an offence under sections 7[4][a] or section 7[5] or section 8[2] or section 8[3] or section 9[2] or section 11[4] or section 12[3] or section 12[6] section 12[7] or section 12[8][a] or section 12[8][b] or section 14[1] or section 15[1] or section 15[14] is liable, on conviction, to imprisonment for 6 years and a fine of R250.000 and, if the offence is continued after conviction, is liable to a further fine of R5.000 for each day during which the offence is so continued. [3] Where an offence specified in subsection [2] is continued beyond a period of one year after conviction by the person so convicted the person is guilty of an offence and liable on conviction to imprisonment for not less than 2 years and not exceeding 7 years.
```

Comment:
Please replace `sections 7[4][a]` with `section 7[4][a]`. Also, please replace `12[6] section` with `12[6] or section`. Furthermore, please replace `50.000`, `5.000`, `250.000` and `5.000` with `50,000`, `5,000`, `250,000` and `5,000` respectively.